plugins/SegmentEditor/stylesheets/segmentation.less

```diff
@@ -366,10 +367,16 @@ html.comparisonsDisabled .segmentationContainer ul.submenu {
 }
 
 .segmentEditorPanel.expanded .add_new_segment {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
Why do we need flexbox for an anchor tag? Seems to work the same without these.

Reply:
The purpose of using flexbox here is to ensure the label will be vertically centered, even when there is only a single line of text.

The difference is small, but here how it is aligned without flexbox:
<img width="220" height="71" alt="Screenshot 2025-11-03 at 11 09 58" src="https://github.com/user-attachments/assets/0d3341e2-463d-46f9-b5b5-c38764fc60ec" />


It also a protection if someone in the future make the button bigger or reduce the inner padding.

Why are you not liking using flexbox in this case?

Reply:
Yes. It would be better to apply it globally.
But I find difficult for now to have a PR that touch every pages of the app. It explodes the time to have it merge and review. 😞